tests/suites/0_stateless/02_function/02_0045_conditional_function_in.sql

```diff
@@ -0,0 +1,17 @@
+SELECT number from numbers_mt(10) WHERE number+1 IN (2, 3 ,20) ORDER BY number;
```

Comment:
Cool, 
I think `tests/suites/0_stateless/02_0045_function_in.sql` can be removed now?

Reply:
sorry, I forget to delete it ...

Reply:
fix it.